app/workers/articles/update_page_views_worker.rb

```diff
@@ -0,0 +1,25 @@
+module Articles
+  class UpdatePageViewsWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority,
+                    lock: :until_executing,
+                    on_conflict: :replace,
+                    retry: 10
+
+    def perform(create_params)
+      PageView.create(create_params)
+
+      article = Article.find(create_params["article_id"])
```

Comment:
same thing about not finding the article and retrying a job for an article that doesn't exist

Reply:
See long comment above.